deep_quoridor/src/agents/dexp.py

```diff
@@ -75,9 +74,15 @@ def __init__(
             self.params = DExpPlayParams.from_str(agent_params_str)
         super().__init__(**kwargs)
 
+    def name(self):
+        return "dexp"
+
```

Comment:
So we don't have to deal with the problem of extending classes having a different name, but actually I hope to be able to remove the pretrained classes and make it work automatically